recipes/dttlib/meta.yaml

```
@@ -0,0 +1,71 @@
+{% set name = "dttlib" %}
+{% set version = "0.3.5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://git.ligo.org/cds/software/cr_tools/dttlib/-/archive/{{ version }}/{{ version }}.tar.gz
+  sha256: 3529939797ec31d32afa30f94633c225988dbc0548b6ad1b882c85da78ea6215 
+
+build:
+  skip: true  # [win or osx]
+  error_overlinking: true
+  error_overdepending: true
+  number: 0
+
+requirements:
+  build:
+   - {{ compiler('c') }}
+   - {{ stdlib("c") }}
+   - {{ compiler('cxx') }}
+   - python
+   - clang
+   - fftw 
+   - gcc
+   - gds-sigp 
+   - glib
+   - libcds 
+   - libclang
+   - make  # [not win]
+   - pip
+   - pkg-config  # [not win]
```

```suggestion
   - make 
   - pip
   - pkg-config
```
`make` and `pkg-config` now support windows, so there is no reason to place this restriction on them.

---

The version tags are removed in the latest commit.